### PR TITLE
Feature/291 update aql terminology grammar with sec

### PR DESCRIPTION
**Pull request by lmarcoruiz**
## Changes

Adapts parser to the latest openEHR SEC agreements.
Implements Validate operation with external terminology server.

## Related issue

Closes https://github.com/ehrbase/project_management/issues/291

### CODE REVIEW

File: service/src/main/java/org/ehrbase/service/FhirTerminologyServerR4AdaptorImpl.java [hunk 1]
```diff
@@ -147,6 +147,7 @@ public final DvCodedText lookUp(final String conceptId) {
 	@Override
 	public final Boolean validate(final DvCodedText concept, final String valueSetId) {
 		// TODO Auto-generated method stub
+		System.out.println("inside the validate method of R4 implemetntation");
```
Comment (chevalleyc) [inline service/src/main/java/org/ehrbase/service/FhirTerminologyServerR4AdaptorImpl.java:150]: I would suggest to use a logger and use a log.debug(...) instead

Comment (Geisterfalle) [reply service/src/main/java/org/ehrbase/service/FhirTerminologyServerR4AdaptorImpl.java:150]: As Luis is absent for some time, could you please do the changed and merge?

Comment (chevalleyc) [reply service/src/main/java/org/ehrbase/service/FhirTerminologyServerR4AdaptorImpl.java]: Done

**Comment by jakesmolka** [comment]
FYI: @lmarcoruiz @chevalleyc those changes to the service pom.xml you just merged are unclean (different formatting) and are creating merge conflicts, see this direct link:
https://github.com/ehrbase/ehrbase/pull/268/files#diff-15f45f0e05aff8eda200826913a1367e

I'll try to revert it, while keeping possible real changes via piggybacking on #267

**Comment by chevalleyc** [comment]
I didn't look at the pom.xml TBH, just integrated the change I suggested. Sorry about that

**Pull request merged**